api/api/authentication.py

```diff
@@ -89,83 +84,7 @@ def check_user(user: str, password: str, required_scopes=None) -> Union[dict, No
 
 # Set JWT settings
 JWT_ISSUER = 'wazuh'
-JWT_ALGORITHM = 'ES512'
-_private_key_path = os.path.join(SECURITY_PATH, 'private_key.pem')
-_public_key_path = os.path.join(SECURITY_PATH, 'public_key.pem')
-
-
-def get_keypair(curve: ec.EllipticCurve) -> Tuple[str, str]:
-    """Generate key files to keep safe or load existing public and private keys.
-
-    Parameters
-    ----------
-    curve : ec.EllipticCurve
-        Elliptic curve type.
-
-    Returns
-    -------
-    private_key : str
-        Private key.
-    public_key : str
-        Public key.
-
-    Raises
-    ------
-    WazuhInternalError(6003)
-        If there was an error trying to load the JWT secret.
-    """
-    try:
-        if not os.path.exists(_private_key_path) or not os.path.exists(_public_key_path):
-            private_key, public_key = change_keypair(curve)
-            try:
-                os.chown(_private_key_path, wazuh_uid(), wazuh_gid())
-                os.chown(_public_key_path, wazuh_uid(), wazuh_gid())
-            except PermissionError:
-                pass
-            os.chmod(_private_key_path, 0o640)
-            os.chmod(_public_key_path, 0o640)
-        else:
-            with open(_private_key_path, mode='r') as key_file:
-                private_key = key_file.read()
-            with open(_public_key_path, mode='r') as key_file:
-                public_key = key_file.read()
-    except IOError:
-        raise WazuhInternalError(6003)
-
-    return private_key, public_key
-
-
-def change_keypair(curve: ec.EllipticCurve) -> Tuple[str, str]:
-    """Generate key files to keep safe.
-    
-    Parameters
-    ----------
-    curve : ec.EllipticCurve
-        Elliptic curve type.
-    
-    Returns
-    -------
-    private_key : str
-        Private key.
-    public_key : str
-        Public key.
-    """
-    key_obj = ec.generate_private_key(curve)
-    private_key = key_obj.private_bytes(
-        encoding=serialization.Encoding.PEM,
-        format=serialization.PrivateFormat.PKCS8,
-        encryption_algorithm=serialization.NoEncryption()
-    ).decode('utf-8')
-    public_key = key_obj.public_key().public_bytes(
-        encoding=serialization.Encoding.PEM,
-        format=serialization.PublicFormat.SubjectPublicKeyInfo
-    ).decode('utf-8')
-    with open(_private_key_path, mode='w') as key_file:
-        key_file.write(private_key)
-    with open(_public_key_path, mode='w') as key_file:
-        key_file.write(public_key)
-
-    return private_key, public_key
+JWT_ALGORITHM = 'ES256'
```

Comment:
Since this PR objective is related to the JWTs behavior, we could use it to unify the JWT-related constants from the APIs in a single file.